docs/containers.config.5.md

```diff
@@ -0,0 +1,91 @@
+% containers.conf(5) Container runtime tools configuration file
+
+# NAME
+containers.conf - configuration file for the container runtime tools like Podman and Buildah
+
+# DESCRIPTION
+The containers configuration file specifies all of the available configuration options and command-line flags for container runtime tools like Podman & Buildah, but in a TOML format that can be easily modified and versioned.
```

Comment:
The containers configuration file specifies all of the available configuration options and command\-line flags for container runtime tools like Podman \& Buildah, but in a TOML format that can be easily modified and versioned.